src/AdminTab/TableList.tsx

```diff
@@ -29,7 +29,7 @@ export const TableList: FunctionComponent = () => {
 
   return (
     <div className="TableList">
-      <Table dataSource={tableList} loading={tableListIsLoading} rowKey="name">
+      <Table dataSource={tableList} loading={tableListIsLoading} rowKey="key">
```

Comment:
why this change?

Reply:
I'm confused why tables have both `key` and `name`? Given the presence of a `key` prop, I assume that will be our primary identifier. Should we drop it?

Reply:
tbh, I didn't dig deep. I recognize this `rowKey` addition from a recent PR of @cristianberneanu so I thought I'd make sure this change is not a leftover of some sort.

Now I looked, and indeed we have a `key` field, possibly specifically for this and I guess if `rowKey=name` does what I think it does then we might drop it instead.